src/actions/tags.js

```diff
@@ -51,7 +51,7 @@ export function createTagInCommunity (params, slug) {
   return {
     type: CREATE_TAG_IN_COMMUNITY,
     payload: {api: true, params, path: `/noo/community/${slug}/tag/`, method: 'POST'},
-    meta: {slug, optimistic: true}
+    meta: {slug, tag: params, optimistic: true}
```

Comment:
If `params` is just a tag object please rename it as such -- it's pretty confusing to figure out what `params` is supposed to be here; maybe you can destructure the argument so people looking at this code know what keys to expect in it?